hubble-fe/src/components/graph-management/GraphManagementList.tsx

```diff
@@ -422,18 +397,26 @@ const GraphManagementListItem: React.FC<
           访问
         </Button>
         <Dropdown.Button
-          options={enabled ? dropdownList : disabledDropdownList}
+          overlay={enabled ? (
+            <Menu onClick={handleDropdownClick(index)}>
+              <Menu.Item key="edit" >编辑</Menu.Item>
+              <Menu.Item key="delete">删除</Menu.Item>
+            </Menu>
+          ) : ( <Menu onClick={handleDropdownClick(index)}>
+          <Menu.Item key="edit" disabled>编辑</Menu.Item>
+          <Menu.Item key="delete">删除</Menu.Item>
+        </Menu>)}
           title="更多"
           size="medium"
           trigger={['click']}
           width={78}
-          onHandleMenuClick={handleDropdownClick(index)}
+          // onClick={handleDropdownClick(index)}
```

Comment:
unused?

Reply:
The exposed parameters of the two components are different, and some need to be rewritten. Part of the commented code is not needed, I will format and delete it later.